documentation/sidebars.js

```diff
@@ -10,128 +10,125 @@
-          id: 'libraries/nodejs/examples',
-          label: 'Examples'
+          type:'doc',
+          label: 'Java for Android',
+          id:'getting_started/java_for_android'
         },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Explanations',
+      items:[
         {
           type: 'doc',
-          id: 'libraries/nodejs/api_reference',
-          label: 'API Reference'
-        },
+          label: 'The Library in a Nutshell',
+          id: 'explanations/nutshell',
+        }
       ]
     },
     {
       type: 'category',
-      label: 'Python',
-      items: [
+      label: 'Examples',
+      items:[
         {
-          type: 'doc',
-          id: 'libraries/python/getting_started',
-          label: 'Getting Started'
+          type:'doc',
+          label: 'Rust',
+          id:'examples/rust'
         },
         {
-          type: 'doc',
-          id: 'libraries/python/examples',
-          label: 'Examples'
+          type:'doc',
+          label: 'Node.js',
+          id:'examples/nodejs'
         },
         {
-          type: 'doc',
-          id: 'libraries/python/api_reference',
-          label: 'API Reference'
+          type:'doc',
+          label: 'Python',
+          id:'examples/python'
+        },
+        {
+          type:'doc',
+          label: 'Java',
+          id:'examples/java'
         },
       ]
     },
     {
-      type: 'category',
-      label: 'Java',
-      items: [
+      type:'category',
+      label: 'Reference',
+      items:[
         {
-          type: 'doc',
-          id: 'libraries/java/getting_started',
-          label: 'Getting Started'
+          type:'doc',
+          label: 'Library Specifications',
+          id:'reference/specifications'
         },
         {
-          type: 'doc',
-          id: 'libraries/java/android_development',
-          label: 'Android'
+          type:'doc',
+          label: 'Rust API',
+          id:'reference/rust'
         },
         {
-          type: 'doc',
-          id: 'libraries/java/examples',
-          label: 'Examples'
+          type:'doc',
+          label: 'Node.js API',
+          id:'reference/nodejs'
         },
         {
-          type: 'doc',
-          id: 'libraries/java/api_reference',
-          label: 'API Reference'
+          type:'doc',
+          label: 'Python API',
+          id:'reference/python'
+        },
+        {
+          type:'doc',
+          label: 'Java API',
+          id:'reference/java'
         },
       ]
+    },
+    {
+      type: 'doc',
+      id: 'troubleshooting',
+      label: 'Troubleshooting'
+    },
+    {
+      type: 'doc',
+      id: 'contribute',
+      label: 'Contribute',
     }
-    ]
-  },
-  {
-    type: 'doc',
-    id: 'specification',
-    label: 'Specification',
-  },
-  {
-    type: 'doc',
-    id: 'troubleshooting',
-    label: 'Troubleshooting'
-  },
-  {
-    type: 'doc',
-    id: 'contribute',
-    label: 'Contribute',
-  }
   ]
 };
```

Comment:
```suggestion
};

```

New line. Please. I can't see this GitHub symbol for missing new lines anymore :trollface: